### ISSUE TITLE
Pick List, Pick List Group and Pick List Item - Ensure images provide informative alternative text - (2036570123)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5598]
### Module: 
 24 Pick List, Pick List Group and Pick List Item 

### Violation: 
 Ensure images provide informative alternative text 

![image](https://user-images.githubusercontent.com/5023024/202742361-bb05f234-146f-4844-8d3b-02ffd3b0a88a.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570123 

### Severity: 
 10 

### Description: 
 [Issue]
There are meaningful images without a textual equivalent. Examples include:
- selection icon beside the text in the "Select multiple items from list" section.

[User Impact]
Screen reader users will be unable to determine what these images represent.

[Code Reference] 
 ```
<calcite-icon aria-hidden="true" icon="check" scale="s" calcite-hydrated=""></calcite-icon>
(...)
<svg class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 12L2 8.689l.637-.636L5.5 10.727l8.022-7.87.637.637z"></path></svg>
 ``` 

### Note: 
 [Suggestion]
Ensure images provide informative alternative text. Textual equivalents must be both concise and descriptive. In this case remove aria-hidden="true" from the ``` <Calcite> ``` element. In the case of svg elements, it is recommended to add role="img", ``` <title> ``` with the text equivalent and aria-labelledby set to the id of the title element

[Compliant Code Example] 
 ```
<calcite-icon icon="check" scale="s" calcite-hydrated=""></calcite-icon>
<svg role="img" class="svg" fill="currentColor" height="100%" viewBox="0 0 16 16" width="100%" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 12L2 8.689l.637-.636L5.5 10.727l8.022-7.87.637.637z" aria-labelledby="title"></path>
<title id="title">Selected </title>
....
</svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
We can ignore this one for now and direct users to use the list component

**Comment by geospatialem** [comment]
Mitigated in a PR prior to the 1.0.5 release, where the svg uses `aria-hidden="true"`.
